internal/provider/provider_test.go

```diff
@@ -22,7 +22,7 @@ func testAccPreCheck(t *testing.T) {
 	// function.
 
 	// Check environmental variables
-	ev := []string{"MERAKI_DASHBOARD_API_KEY"}
+	ev := []string{"MERAKI_DASHBOARD_API_KEY, TF_ACC_MERAKI_MX_LICENCE, TF_ACC_MERAKI_MX_SERIAL, TF_ACC_MERAKI_ORDER_NUMBER"}
```

Comment:
Great job referencing TF_ACC vars. Please add these to the repository secrets under the settings page of this repo so that our GitHub Action Runner can perform the acceptance tests.

Reply:
ok added.